Scoop/View/LabeledTextField.swift

```diff
@@ -53,7 +53,7 @@ class LabeledTextField: UIView {
     private func setupLabel(title: String) {
         nameLabel.text = " \(title) "
         nameLabel.font = .systemFont(ofSize: 12)
-        nameLabel.textColor = .black
+        nameLabel.textColor = .mutedGrey
```

Comment:
Yea you get the point LOL. Just make sure you're using a custom color. I would refactor the entire codebase and make it use `UIColor.scoop.mutedGrey` to make it more clear that it is a custom color.